src/common/SurgePatch.cpp

```diff
@@ -1221,6 +1240,26 @@ unsigned int SurgePatch::save_patch(void **data)
     return psize;
 }
 
+Parameter *SurgePatch::parameterFromStorageName(std::string stName)
+{
+    auto it = param_ptr_by_storagename.find(stName);
+    if (it != std::end(param_ptr_by_storagename))
+    {
+        return it->second;
+    }
+    else
+    {
+        return nullptr;
+    }
+}
+Parameter *SurgePatch::parameterFromOSCName(std::string stName)
```

Comment:
Likewise, this should take a std::string_view.

Reply:
See comment on parameterFromStorageName.